README.md

```diff
@@ -126,39 +131,27 @@ By design Services should never expose DB Entities outside. All communication wi
 [Mapstruct](https://mapstruct.org/) is advised to use for mapping Entities to DTOs and vice versa.
 
 Typical Service & DTO pattern:
-```java
-    @Override
-    public ExampleEntityFirstDto save(ExampleEntityFirstDto exampleEntityFirstDto) {
-        ExampleEntityFirst exampleEntityFirst = exampleEntityFirstMapper.toEntity(exampleEntityFirstDto);
-        exampleEntityFirst = exampleEntityFirstRepository.save(exampleEntityFirst);
-        return exampleEntityFirstMapper.toDto(exampleEntityFirst);
-    }
-```
-
-### CI/CD
```

Comment:
why this block was removed?

Reply:
README.md is not updated yet
ignore it for now

Reply:
returned